experimental/dds/tree2/src/core/rebase/utils.ts

```diff
@@ -276,7 +284,10 @@ export function rebaseChange<TChange>(
 
 	const changeRebasedToRef = sourcePath.reduceRight(
 		(newChange, branchCommit) =>
-			changeRebaser.rebase(newChange, inverseFromCommit(changeRebaser, branchCommit)),
+			changeRebaser.rebase(
+				newChange,
+				inverseFromCommit(changeRebaser, branchCommit, repairData),
```

Comment:
Nit: might as well pass `repairData?.get(branchCommit.revision)` to reduce the amount of code that has to know about the map.

Reply:
Done in latest commit.